config/deploy/web.yaml.erb

```diff
@@ -5,6 +5,7 @@ metadata:
   annotations:
     shipit.shopify.io/restart: 'true'
 spec:
+  replicas: <%= environment == 'production' ? 3 : 2 %>
```

Comment:
Specifying a replica and HPA policy are exclusive, they will try to overwrite one another